app/webpacker/styles/lead_providers.scss

```diff
@@ -0,0 +1,537 @@
+.index-group-lead-provider {
+  background: govuk-colour("light-grey");
+  padding: 1.5em;
+  margin-bottom: 1.5em;
+}
+
+
+.app-list--definition {
```

Comment:
This might not apply due to the other comments, but BEM components should go in their own files, not a page-based file like this

Reply:
Deleted anything not used